bindings/azure/blobstorage/blobstorage.go

```diff
@@ -205,43 +119,45 @@ func (a *AzureBlobStorage) Operations() []bindings.OperationKind {
 }
 
 func (a *AzureBlobStorage) create(ctx context.Context, req *bindings.InvokeRequest) (*bindings.InvokeResponse, error) {
-	var blobHTTPHeaders azblob.BlobHTTPHeaders
-	var blobURL azblob.BlockBlobURL
 	var blobName string
 	if val, ok := req.Metadata[metadataKeyBlobName]; ok && val != "" {
 		blobName = val
 		delete(req.Metadata, metadataKeyBlobName)
 	} else {
 		blobName = uuid.New().String()
```

Comment:
Please use `uuid.NewRandom()` which also returns an error. `uuid.New()` can panic if there's not enough entropy.